Cpp/odin/odin.cpp

```diff
@@ -41,7 +41,7 @@ const fostlib::setting<fostlib::string> odin::c_facebook_secret(
 const fostlib::setting<fostlib::string> odin::c_facebook_endpoint(
     "odin/odin.cpp", "odin", "Facebook API Endpoint", "https://graph.facebook.com", true);
 
-const fostlib::setting<fostlib::string> odin::c_google_aud(
+const fostlib::setting<fostlib::json> odin::c_google_aud(
     "odin/odin.cpp", "odin", "Google", "", true);
```

Comment:
Almost certainly you want the default value to be an empty JSON not an empty string. Use `fostlib::null`